examples/app-with-service-account/README.md

```diff
@@ -0,0 +1,64 @@
+# Grafana app plugin with Service Account integration
+
+This plugin is an example of how to integrate Service Account authentication into a Grafana plugin.
+
+**Note:** This plugin requires Grafana 10.2 or later and the `externalServiceAccounts` feature toggle must be enabled. This is an experimental feature.
+
+## How to use
+
+This app allows you to create a service account in Grafana tailored to your plugin needs. Grafana will provide the plugin with a service account token that you can use to request the Grafana API.
+
+![screenshot](./src/img/screenshot-showcase.png)
+
+## Authentication flow
+
+The plugin uses a [Grafana service account token](https://grafana.com/docs/grafana/latest/administration/service-accounts/#service-account-tokens) to authenticate against the Grafana API. To enable it, add the section below to your `plugin.json` file.
+
+```json
+  "externalServiceRegistration": {
```

Comment:
Do you know if this if documented somewhere? Could not find in the portal or grafana.com/docs.

Reply:
I'll tackle that soon :)